src/main/java/org/example/homework9/mobilePhoneApp/MobilePhone.java

```diff
@@ -0,0 +1,58 @@
+package org.example.homework9.mobilePhoneApp;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MobilePhone {
+    private String myNumber;
+    private List <Contact> myContacts;
+
+    public MobilePhone(String myNumber, List<Contact> myContacts) {
+        this.myNumber = myNumber;
+        this.myContacts = new ArrayList<>();
+    }
+
+    public boolean addNewContact(Contact contact) {
+        return myContacts.add(contact);
+    }
+
+    public boolean updateContact(Contact oldContact, Contact newContact) {
+        int index = findContact(oldContact);
+        if (index >= 0) {
+            myContacts.add(index, newContact);
```

Comment:
Using add here is in correct, please read the Java doc. You should substitute the contact